plugin/gcpsample_asset_manager/GCPSampleAssetManagerInterface.py

```diff
@@ -67,18 +67,13 @@ def initialize(self, managerSettings, hostSession):
             )
         
         # initialize a client for Google Cloud Spanner
-        print("Initializing Google Cloud Spanner for our OAIO plugin")
+        hostSession.logger().debug("Initializing Google Cloud Spanner for our OAIO plugin")
```

Comment:
The logging mechanism makes sure these get back to the user in the most appropriate fashion.